.ado/publish.yml

```diff
@@ -5,38 +5,156 @@
     displayName: Pack VSCode Extension
     workingDirectory: '$(System.DefaultWorkingDirectory)/vscode'
+    condition: eq(variables['Agent.OS'], 'Darwin')
+
+  - script: |
+      ls vscode/*
+      ls vscode/out/*
+    displayName: List VSCode Extension
+    condition: eq(variables['Agent.OS'], 'Darwin')
+
+  - publish: $(System.DefaultWorkingDirectory)/vscode/qsharp-vscode-$(Ver.Major).$(Ver.Minor).$(Ver.Patch).vsix
+    artifact: VSIX
+    displayName: Upload VSCode Extension Artifact
+    condition: eq(variables['Agent.OS'], 'Darwin')
+
+  - publish: $(System.DefaultWorkingDirectory)/vscode/out/extension.js
+    artifact: VSIX.EntryPoint
+    displayName: Upload VSCode Ext EntryPoint Artifact
+    condition: eq(variables['Agent.OS'], 'Darwin')
+
+  - script: |
+      dir target\wheels\*
+    displayName: List Py Packages on Win
+    condition: eq(variables['Agent.OS'], 'Windows_NT')
+
+  - script: |
+      ls target/wheels/*
+    displayName: List Py Packages on non-Win
+    condition: ne(variables['Agent.OS'], 'Windows_NT')
+
+  - publish: $(System.DefaultWorkingDirectory)/target/wheels
+    artifact: Wheels.Mac
+    displayName: Upload Python Artifacts Mac
+    condition: eq(variables['Agent.OS'], 'Darwin')
+
+  - publish: $(System.DefaultWorkingDirectory)/target/wheels
+    artifact: Wheels.Win
+    displayName: Upload Python Artifacts Win
+    condition: eq(variables['Agent.OS'], 'Windows_NT')
+
+
+- job: "Publish"
+  dependsOn: Build
+  pool: 
+    vmImage: 'windows-latest'
+
+  steps:
+  - download: current
+    artifact: Wheels.Win
+    displayName: Download Python Artifacts Win
+
+  - download: current
+    artifact: Wheels.Mac
+    displayName: Download Python Artifacts Mac
+
+  - script: |
+      mkdir target\wheels
+      move ..\Wheels.Win\*.whl target\wheels
+      move ..\Wheels.Mac\*.whl target\wheels
+      dir target\wheels\*
+    displayName: Move Py Artifacts to Publishing Dir
+
+  - download: current
+    artifact: VSIX
+    displayName: Download VSCode Extension Artifact
+
+  - download: current
+    artifact: VSIX.EntryPoint
+    displayName: Download VSCode Ext EntryPoint Artifact
+
+  - script: |
+      md vscode\out
+      move ..\VSIX.EntryPoint\extension.js vscode\out
+      dir vscode\out\*
+      move ..\VSIX\*.vsix vscode
+      dir vscode\*
+    displayName: Move VSIX to Publishing Dir
+
+  - script: |
+      npm install -g @vscode/vsce
+    displayName: Install Prereqs for VSCode Ext Publishing
+
+  - task: EsrpRelease@2
+    inputs:
+     ConnectedServiceName: 'ESRP_Release'
+     Intent: 'PackageDistribution'
+     ContentType: 'PyPi'
+     PackageLocation: '$(System.DefaultWorkingDirectory)/target/wheels'
+     Owners: '$(OwnerPersonalAlias)@microsoft.com'  # NB: ESRP corrected from group email `qdevtools` to personal one.
+                                                    #     Group email here fails the task with non-actionable output.
+     Approvers: 'billti@microsoft.com'
```

Comment:
Can this be a group we create rather than a specific individual? That's easier to maintain as people change roles.

Reply:
With group email the task fails with this non-actionable output (I reported this to ESRP):
```
Error while submissing request to Release Gateway 
Execution failed with error :- 
CorrelationId: 68d72e92-f4c2-4696-97b7-2362dafcae2d
400--Bad Request
##[error]Task failed, Please see the error details and reach out to ESRP Release DRI via IcM(https://aka.ms/CreateIncidentOnESRPRelease) for further assistance.
```